src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -94,7 +94,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     String lectureName = request.getParameter(PARAM_NAME);
     Entity lectureEntity = LectureUtil.createEntity(lectureName, videoUrl, videoId.get());
     datastore.put(lectureEntity);
-    initializeTranscript(lectureEntity);
+    try {
+      initializeTranscript(lectureEntity);
+    } catch (IOException | ServletException e) {
+      // If there was an error initializing the transcript, then this lecture won't have one.
```

Comment:
Can you add a unit test for sending a video without a transcript?

Reply:
^^

Reply:
Added!